common/src/chain/block/block_index.rs

```diff
@@ -0,0 +1,61 @@
+use crate::chain::block::Block;
+use crate::chain::ChainConfig;
+use crate::primitives::{BlockHeight, Id, Idable};
+// use crate::Uint256;
+use parity_scale_codec::{Decode, Encode};
+
+#[derive(Debug, Clone, PartialEq, Eq, Encode, Decode)]
+#[allow(dead_code, unused_variables)]
+pub struct BlockIndex {
+    block_id: Id<Block>,
+    prev_block_id: Option<Id<Block>>,
+    // TODO: When Carla finish her code, we should use Uint256 at the moment it's unable to store to DB
+    //  pub chain_trust: Uint256,
```

Comment:
Is this comment still valid?

Reply:
I saw your comment and I was coming to it. This is a minor issue as chain trust is still not that big of a deal when we're just adding one to it for every new block. Once we use proper PoW/PoS, we'll have to use H256/Uint256 for it. But it doesn't matter right now. The comment is valid in the sense that we have to change it to 256-bit number only.